bin/server.py

```diff
@@ -0,0 +1,130 @@
+# Copyright (C) 2020 Dmitry Ivanko. All Rights Reserved.
+# Copyright (C) 2020 Vladislav Yarovoy. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Main module of cusdeb-helpik"""
+import json
+import logging
+
+import aiohttp_cors
+import requests
+from aiohttp import web
+from aiohttp.web_request import Request
+from aiohttp_cors import CorsViewMixin
+from bs4 import BeautifulSoup
+
+from bin import config
+
+logging.basicConfig(filename='cusdeb-helpik.log', level=logging.INFO)
```

Comment:
Make it possible to configure the path to the log file via an environment variable.